src/cpdetect/compiler_detector/compiler_detector.cpp

```diff
@@ -434,7 +434,7 @@ ReturnCode CompilerDetector::getAllSignatures()
 					{
 						toolInfo.addTool(
 							DetectionMethod::YARA_RULE,
-							metaToStrength(rule.getMeta("language")),
+							metaToStrength(rule.getMeta("strength")),
```

Comment:
This is probably just a bug fix of code from #746 and is not related to the detection of PyInstaller, right?

Reply:
Yes.